docs/source/tutorials/advanced/index.rst

```diff
@@ -0,0 +1,9 @@
+Advanced tutorial
+=================
+
+.. toctree::
+   :maxdepth: 3
+
+   semi_sl
+   backbones
+   datumaro
```

Comment:
This is an optional comment for a future reference. Once we have enough material, we might need to categorize these into something like data, model learning-related material.